crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -15,6 +15,20 @@
 #
 
 class Api::UpgradeController < ApiController
+  # disable upgrade API until upgrade to next version is implemented
+  before_action do
+    # skip filter if we're in the middle of upgrade from previos version
+    unless File.exist?("/var/lib/crowbar/upgrade/7-to-8-upgrade-running")
+      render json: {
+        errors: {
+          unexpected_error: {
+            data: "Upgrade not supported in this version"
```

Comment:
Just a detail, but it might be nice to have the message better, at least by saying "it is not _yet_ supported".